### PR TITLE
Fixes #151

### LINKED ISSUE #151: Thoughts on the "Flawed gems" section
I think we should get rid of it. Instead of negative recommendations, let's add positive ones.

* A negative list is hard to mantain. A positive list too, but it's definitely easier to list the stuff one considers good, than everything one considers bad. For example, if there was a debugging section, one would say here "don't use `ruby-debug`, don't use `debugger` either" where one can just say "use `byebug`".
* A negative list is too subjective. For example, I consider `minitest-autotest` (`autotest` successor) to be as good as `guard`.
* A negative list doesn't have a good vibe. Let's focus on good stuff. Gems like `guard` or `simplecov` only get mentioned in this section, let's encourage their use instead of discouraging the alternatives.

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
@katgironpe @bbatsov Thoughts?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1096,30 +1096,10 @@ your application.
   same gem versions when they do a `bundle install`.
 <sup>[[link](#gemfile-lock)]</sup>
 
-## Flawed Gems
+## Awesome Gems
```
Comment (bbatsov) [inline README.md]: Just kill the section altogether as I have an aversion towards "awesome" lists.

**Comment by bbatsov** [comment]
The commit message can be a bit more descriptive (and in sync with the project general commit message style).

**Comment by katgironpe** [comment]
Agree with @bbatsov. Simply remove the section.

File: README.md (hunk 1 above)
Comment (deivid-rodriguez) [inline README.md]: :+1:

**Comment by deivid-rodriguez** [comment]
@bbatsov What's the general commit message style? I see some commit messages in present tense, some in past tense; some capitalized, some not; some ending with a dot, some not...

**Comment by bbatsov** [comment]
Hmm, guess I didn't write a very extensive contributing.md. :-)

On all my projects I follow these rules http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

**Comment by deivid-rodriguez** [comment]
Rebased it. Do you want me to rename the branch so the merge commit is clearer?

**Comment by bbatsov** [comment]
That will do. Thanks!

**Comment by deivid-rodriguez** [comment]
Thanks to you! :smiley:

**Pull request merged**